_posts/0000-01-02-umbreboy.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
There is only one god and his name is Death, and there is only one thing we say to Death: ‘Not today.